content/id/docs/concepts/overview/working-with-objects/names.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Nama
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+Seluruh objek di dalam REST API Kubernetes secara jelas diidentifikasi dengan nama dan UID.
+
+Atribut disediakan untuk pengguna yang tidak unik, Kubernetes menyediakan [labels](/docs/user-guide/labels) dan [anotasi](/docs/concepts/overview/working-with-objects/annotations/).
+
+Lihat pada [dokumentasi desain pengidentifikasi](https://git.k8s.io/community/contributors/design-proposals/architecture/identifiers.md) untuk aturan sintaks yang tepat untuk Nama dan UID.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Nama
+
+{{< glossary_definition term_id="name" length="all" >}}
+
+Dengan ketentuan, nama dari *resources* Kubernetes bisa mencapai panjang maksimal 253 karakter dan terdiri dari karakter alfanumerik huruf kecil, `-`, dan `.`, tetapi *resources* tertentu punya lebih banyak batasan yang spesifik
```

Comment:
Rephrase -> Berdasarkan ketentuan, nama dari _resources_ Kubernetes memiliki panjang maksimum 253 karakter yang terdiri dari...

Please change `*` with `_` to make the character italic.